course_flow/views.py

```diff
@@ -1205,6 +1205,29 @@ def import_view(request):
     return render(request, "course_flow/import.html")
 
 
+class SALTISEAnalyticsView(
```

Comment:
Love the class-based views.  Is this the kind of admin you wanted (i.e. not really an admin site, just group-protected views)?  It seems like it does the job, but we can do the other kind too by creating a custom AdminSite [https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#multiple-admin-sites-in-the-same-urlconf].

Reply:
Yeah I think this is all I wanted, just a group-protected thing for saltise staff to be able to do a few small things